i-RIC/prepost-gui
# Close #869, #893, #894

Please test with the following steps:

# Issue 869

## Test importing

1. Start a new project for Nays2DH
2. Select "Geographic data" -> "Elevation (m)"
3. Select "Import" from right-clicking menu
4. Select "test.png" and import it.
5. Dialog to select coordinate system for the geodata and project are shown. Select "WGS 84".
6. Make sure that the file is imported successfully, 

## Test exporting

1. Select the "Raster data1", and select "Export" from right-clicking menu.
2. Select Grayscale 16bit PNG files (\*.png), input filename, and click on "Save" button.
3. Make sure that \*.png, \*.pgw, \*.png.meta are output, and they are the same to the files imported.

# Issue 893 

1. Select the "Raster data1", and select "Export" from right-clicking menu.
2. Select "GeoTiff files (\*.tif)" or "ArcInfo ASCII files (\*.asc)", input filename, and click on "Save" button.
3. Make sure that a file is exported successfully.

# Issue 894

1. Select the "Raster data1", and select "Export" from right-clicking menu.
2. Select "NetCDF files (\*.nc)", input filename, and click on "Save" button.
3. Make sure that a file is exported successfully.

[issue-869.zip](https://github.com/i-RIC/prepost-gui/files/5930044/issue-869.zip)
